build: reintroduce stack unwinding, don't abort on panic

simonsan opened the pull request:
We have aborting on panic in our rustic `Cargo.toml`. For `rustic` this may affect the stack traces that `human-panic` spits out. This will make it easier, to find bugs, that are reported to us by the users, having their stack traces. Reintroducing the unwinding code will impact the binary size, which is currently around 8 MiB, and it may go up a few MiBs.

aawsome commented:
@simonsan Do you have numbers how much the binary size increases? I mean I would not judge this important enough for a significant size increase, actually..

simonsan commented:
On Windows:

with `panic = "abort"`: 18.3 MiB
without: 27.7 MB

aawsome commented:
> On Windows:
> 
> with `panic = "abort"`: 18.3 MiB without: 27.7 MB

puh.. that's heavy! I would actually prefer to leave it as "abort"...

simonsan commented:
> > On Windows:
> > with `panic = "abort"`: 18.3 MiB without: 27.7 MB
> 
> puh.. that's heavy! I would actually prefer to leave it as "abort"...

What if we activate `abort`, when rustic is `v1.0.0` and we have fixed most of the errors and squashed bugs. I think at this beta-stage a better stack trace is much more valuable and important than the actual binary size. 😬
